<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarrierWindowClass</class>
 <widget class="QDialog" name="CarrierWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CarrierWindow</string>
  </property>
  <widget class="QLabel" name="window_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New/Edit Carrier</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="coalition_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Coalition :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="coalition">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neutral</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="alias_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Alias :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Select Carrier</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="alias">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="tanker_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tanker :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tanker">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Select Tanker</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="base_frequency">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="base_frequency_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Base Fq :</string>
   </property>
  </widget>
  <widget class="QLabel" name="marshall_frequency_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Marshall Fq :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="marshall_frequency">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="lso_frequency_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LSO Fq :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lso_frequency">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="marshall_relay_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Marshall Relay :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lso_relay">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Select LSO Relay</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="marshall_relay">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Select Marshall Relay</string>
   </property>
  </widget>
  <widget class="QLabel" name="lso_relay_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LSO Relay :</string>
   </property>
  </widget>
  <widget class="QLabel" name="trapsheet_path_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>620</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Trapsheet Path :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="trapsheet_path">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>620</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>C:/airboss-trapsheets</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stat_path_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stat Path :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stat_path">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>620</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>C:/airboss-stats</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>680</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>680</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="menu_mark_zone">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Menu Mark Zone</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="menu_smoke_zone">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Menu Smoke Zone</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="nice_guy">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nice Guy</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="handle_ai">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Handle AI</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="infinite_patrol">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Infinite Patrol</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="single_carrier">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Single Carrier</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="control_area">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="control_area_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Control Area :</string>
   </property>
  </widget>
  <widget class="QLabel" name="recovery_mode_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Recovery Mode :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="recovery_mode">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="max_stacks_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Stacks :</string>
   </property>
  </widget>
  <widget class="QLabel" name="max_patterns_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Patterns :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="max_stacks">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="max_patterns">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="difficulty_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Difficulty :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="difficulty">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Flight Student</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Naval Aviator</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TOPGUN Graduate</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="tacan_morse">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>440</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QComboBox" name="tacan_band">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Y</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="tacan">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>126</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="tacan_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tacan :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="icls_morse">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>470</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="icls_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>470</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Icls :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="icls_channel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>126</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QComboBox" name="recovery_type">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Cyclic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Alpha Strike</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="recovery_type_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Recovery Type :</string>
   </property>
  </widget>
  <widget class="QLabel" name="event_duration_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Event Duration :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="event_duration">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="event_ia_duration_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Event IA Duration :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="event_ia_duration">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>570</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="add_recovery_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>560</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="recoveries_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>540</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Alpha Strike Recoveries : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="rm_recovery_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>590</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QListWidget" name="recoveries">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>560</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
